doc/source/notebooks/manual/advanced/ordinal_regression.ipynb

```diff
@@ -7,7 +7,212 @@
zg5HCNGNIXHBXDA0ho+zy1myvoA/XjrU2SER4mfkXz8dyz83FfLUp4d4Y0cxGw9V8rcrnNdBHp8Wzh8uGcLvVhzk9x8eBPCopOXlzUX8+aNcAK4YncDvL84i0IFFR12FucPKgZI6dh+rZdfRGvYcq6XRbOn2WKNBR6ifF6F+RkL9jAT7emE06PDS6/DSa13fO28bDTriQnxJDvcjOdyfmCAfmZnwHSRhEeIb/ruvjJ1Ha1i+u5Rbp6Q5O5x+Z0N+JTknGogM9OaK0bJ5gRCu6u5ZA1l9sJy3d5awcGo68SG+zg4JnU7j9mkZTM+M4t5lB8g50cC1L23npokp/HJOplO2Y75xQgqAPWkZlhDMqKRQh8fhaG/uKLYnK7+ZO9gjPw+PVDXzxvZjLN9TSn1rxynPxYf4MjYllDEpYQyLDyYi0JtQPy98vfSyyUwfkIRFiG+4cUIKmw9X8fqOY9w8OdVlKjH3B1ab4q+r8wC4ZXKqbHEphAsbGB3IxcPj+PDAcZ5bV8BjPx7m7JDsBscGsWLxJJ5bV8CS9QW8svUoG/JN/O3KEYxNcXwV9RsnpKDRuWPTyMQQh7+/M8wdFsNr249xy+RU5nnQxSeL1cZnuSbe2HGMzYer7I8PjA5gfFo4Y1LCGJMcSpwLJPieRBIWIb5hxqAo4kN8OVbdwto8E7Oyop0dUr/x9q5i8sobiQ/x9ahpE0L0V784bwBHq5uZleXcBe7d8dLruHvWQGZlRXPf8gPklTdy1YvbuHlSKvfPyXT49sc/6Rpp+UpruxVfo3tdlDF3WPHS69DrNEL8jKxcPMljtis2NZh5a2cJb+0sprzBDICPl45LRsRxw/hkhid4RqLqqmTRvRDd+OfGQv7ycR6+Xnr+ce1IzpOk5bQazB1M+9sGaprbWXLdKC4cLovthegPlFIuP4WlzWLlH2sLeGFjIVabYnhCMEuuG+W0bV2P17Vy7Uvb+dnEFG6alOqUGHpbg7mDW1/dTXpUAI9cNtTl20RvKTA1sXRTIf/dV0aHtbNPnBbhz/Xjk7liVALBfp63ZseRerro3jPSZiHO0PxJqfx4VDytHVYWvLabTw6WOzskl1dS04KfUc+5KWHMHRbj7HCEED10csfUauv9i5i9wdug577Zmbx72wQSQn35orSeC5/dzKc5FU6JZ3tRNceqW3h4ZQ7/+vyIU2LoTaZGM9e8uJ0dR2r4LKeCysZvb6/rbvaX1LHwtd3Memojy3aXYrUpZg+J5o1bxrH23qncPDlVkhUXIiMsQnwHpRTPrD3Mqi9PsPy2iQT7yonrdMwdVmpb2okNlrm9QvQnlY1tPL46j7rWDl668bQXO52qvqWDe5fv57NcEwALp6Rx3+xMhxd2fG3bUR5a0blz2A3jk/jdRUMwGvrXdWClFCu/OMHDHx6kprmd1Ah//jP/XLcoSNgdpRSbDlfxzw2FbCuqBjq3Ep43OoEFU9KktowTSB0WIXrJyfOUO6w2rDb1vYvJ39xRzAf7ynj22pHEBDu/toEQQpxOVVMbUx5fT0u7lXdvm8AYJyxsPxNKKV7aXMRfV+djtSnGJIfyj+tGOvxiyfLdJfzmv9m0W22MTArhhetH95vzvqnBzG8/yGZN1yjVxPRwnr12JBEB3k6OrPcdqWpmdXY5K/aXkVfeCECgt4Hrxyczf1IKUS5Qh8hTScIiRC9TSvHg+1+SV97Iyz8dQ0SAN8XVLXywv4wAbwPzJ3fOY84vb2T205uICPDmhRtGOWVHG0faUlDFZ7kV3DVzoAyfC9GPPbkmn2fXFTA2JZRlCyf0izUMu4/WsPjNfZQ3mAnzN/L01ecwZWCkQ2M4UFLHotf3cLzeTGSgN+vundovapU8uiqXpZuKCPA28JsLB3PN2MR+8TfvCaUUuScaWX2wnE+yy8mvaLQ/FxHgzc2TU7l+fBJB/eDv5O4kYRGil1U3tXHJc1soq2slMcyXqEAf9hyrBSA22IctD8xAp9Oobmpj8Zv72FZUjUGn8fuLs7hhfLLbfBCczGK1ceGzn5Nf0civ5w5iwZR0Z4ckhPiBGswdTH18PbUtHfz7prFMH+R6O4d1p7qpjbve2c/mw1VoGswdFsvi6RkMjg1yaAw/f3sf41PDuXPmAIe975my2ZR9q/7mNgt/WHmQu84b2K+36G1pt3C8zszxulZO1LdyqKKJT3MqKK5psR8T6GPgvMHRzB4Sw7TMSNly34VIwiJEHzA1mrnl1d18UVoPgK+XnjlDY7hsZDw/yoiwfxBYrDYe+ziP/+tajHnVmAT+eOlQtztJvr79GL/9IJvEMF8+vXuq2/1+QnialzYV8ciqXAbHBvHRnZP7TR0qq02xZH0B/1h32L7T03mDo7lzRgYjHFQ3xWK1odM0+79ZzvEGksL9CPB2bgUJpRSHKppYc7CcVdnlvHvbBPydHNPplNS0cNjUSKPZQlObhaau743mzq/61vbOJKW+lbqWjm5/Rri/kfOHRDNnaCwT0sL73foiTyEJixB9pKXdwr+3HCU+xJdZWdHfe+L/YF8ZD7z3BW0WGyMSQ3jr1nFOqdTcF+pbO5j+ROc2xi9cP4oLhsk2xkL0d+YOK9Of2MCJejPPXHMOl54T7+yQzsiJ+lZe3FjEWzuLabPYAJgyMJI7Z2Q4dHruifpWLnr2c4J8vbhqTCLnDY4iIyrAYSPtbRYr24tqWJdbwdo8E6W1rfbnHr18GNeNS3JIHD1lajSzrbCarQXVbCmsOiXe0zHqdcSG+BAX7EtsiA8JIb5MyohgTEoY+n6ScHsySViEcBHZZfUsfG0PUwZGulQl6bP15//l8PLnRzg3NYx3Fox3yylvQniid3YV86v3v+S2qek8MGeQs8P5QSob23j58yJe33aM5nYrAONSw1g8I4PJGRF9fr46WtXMrf/ZzWFTk/2xpDA/zhsczXmDoxibGtZru5rZbIqq5jaiAjsXjre0Wxj3yFoa2yz2Y8L9jUwfFMWFw2OZNjDS6edr
```

Comment:
if you're keen I would love to see the probability for a single point as a barplot, next to this one. This is just a single vertical slice, right?